src/org/zaproxy/zap/extension/reveal/ExtensionReveal.java

```diff
@@ -189,7 +188,7 @@ public boolean onHttpResponseReceive(HttpMessage msg) {
 
 	private void revealFields(HttpMessage msg) {
 		boolean changed = false;
-		String response = msg.getResponseHeader().toString() + msg.getResponseBody().toString();
+		String response = msg.getResponseBody().toString();
 		Source src = new Source(response);
```

Comment:
You're probably trying to minimize change I guess, but there's no reason (that I see) for use of response. Could just be: `Source src = new Source(msg.getResponseBody().toString());`

Reply:
No, the `response` is being used in a debug statement (in other places I did that).

Reply:
Ok